data/templates/default.latex

```diff
@@ -63,10 +63,21 @@ $else$
   \setmathfont[$for(mathfontoptions)$$mathfontoptions$$sep$,$endfor$]{$mathfont$}
 $endif$
 $endif$
+  \ifxetex
 $if(CJKmainfont)$
     \usepackage{xeCJK}
     \setCJKmainfont[$for(CJKoptions)$$CJKoptions$$sep$,$endfor$]{$CJKmainfont$}
 $endif$
+  \fi
+  \ifluatex
+$if(luatexjapresetoptions)$
+    \usepackage[$for(luatexjapresetoptions)$$luatexjapresetoptions$$sep$,$endfor$]{luatexja-preset}
+$endif$
+$if(CJKmainfont)$
+    \usepackage[$for(luatexjafontspecoptions)$$luatexjafontspecoptions$$sep$,$endfor$]{luatexja-fontspec}
+    \setmainjfont[$for(CJKoptions)$$CJKoptions$$sep$,$endfor$]{$CJKmainfont$}
+$endif$
+  \fi
 \fi
```

Comment:
Note, this way of doing it gives you
```
\ifxetex
\fi
\ifluatex
\fi
```
when the CJKmainfont and luatexjapresentoptions variables are not set (as is the default).
It would be better to make all of this conditional on CJKmainfont.  (Or you might need two conditionals, one with CJKmainfont, one with luatexjapresetoptions.)

Note also that the writer tests may need updating (though the need for this will probably go away if you make these things conditional, since the writer tests don't set these variables).